### DISCUSSION TITLE
DOSBox Staging Starter Pack

### DISCUSSION THREAD

**Discussion by Grounded0** [dosbox-staging/dosbox-staging#3954] [General]
### Download

https://archive.org/details/dosboxstarterpack

### Issues

If you notice any problems or issues with the current release, please post a comment below, and I'll take a look.

**Comment by Torinde** [comment]
Do you plan to add:
- [ ] SC-55 ROMs
- [ ] #256, 
  - [ ] DOS32/A
  - [ ] dos4gw
  - [ ] [DOS4GU for unbound](https://github.com/MartyShepard/DOSBox-Optionals)
  - [ ] cwsdpmi
- [ ] #604 
- [ ] #603
- [ ] [CD player](https://github.com/dosbox-staging/dosbox-staging/issues/609)
- [ ] [MPXPLAY](http://mpxplay.sourceforge.net/) audio player
- #601
  - [ ] Necromancer's DOS Navigator 
  - [ ] [Doszip Commander](https://github.com/nidud/doszip)
  - [ ] ARJ [developer](https://github.com/joncampbell123/dosbox-x/issues/5882)/LHA/ZIP
- [ ] https://github.com/dosbox-staging/archived-discussions-for-dosbox-staging/discussions/8 
  - [ ] [mTCP](http://www.brutman.com/mTCP/)
  - [ ] ne2000.com
  - [ ] winpkt.com

**Reply by Grounded0** [reply]
I'll add Sound Canvas ROMs once 0.83.0 is released.

I'll also add DOS/32A since many demos don't ship with the required DOS extender.

Thanks for those suggestions.
